GSA/digitalgov.gov
# Cltm archive uswdsblogs 2

## Summary

Archived 4 pages and added redirects to corresponding topic pages.

## Preview

[Link to "3 teams..."](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cltm-archive-uswdsblogs-2/2016/04/06/three-teams-using-the-draft-u-s-web-design-standards-talk-about-their-experiences/)

[link to "making the draft..."](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cltm-archive-uswdsblogs-2/2016/03/16/making-the-draft-u-s-web-design-standards-better-through-your-feedback/)

[link to "how to integrate the draft..."](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cltm-archive-uswdsblogs-2/2016/03/25/how-to-integrate-the-draft-u-s-web-design-standards-into-existing-projects/)

[link to "fractal..."](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cltm-archive-uswdsblogs-2/2017/06/13/fractal-and-federalist-join-the-u-s-web-design-standards/)



## Solution

Provide a summary of the solution this PR offers.




## How To Test

1. Check that all original pages are archived and instead redirect.
2.  Make sure each the correct pages redirect to either the Design topic page or the Open Source topic page:

- https://digital.gov/2016/04/06/three-teams-using-the-draft-u-s-web-design-standards-talk-about-their-experiences/
- https://digital.gov/2016/03/16/making-the-draft-u-s-web-design-standards-better-through-your-feedback/
- https://digital.gov/2016/03/25/how-to-integrate-the-draft-u-s-web-design-standards-into-existing-projects/
- should redirect to: https://digital.gov/topics/design/
- 
- https://digital.gov/2017/06/13/fractal-and-federalist-join-the-u-s-web-design-standards/
- should redirect to: https://digital.gov/topics/open-source/
